src/Components/banner/banner.jsx

```diff
@@ -0,0 +1,51 @@
+import React, {useState, useEffect } from "react";
+import './banner.css'
+
+
+    const images = [
+      "/images/bannerImage.jpg",
+      "/images/banner2.png"
+    ]
+
+
+    function Banner() {
+      const [currentIndex, setCurrentIndex] = useState(0);
+    
+      const moveSlide = (direction) => {
+        if (direction === 'next') {
+          setCurrentIndex((prevIndex) => (prevIndex + 1) % images
+          .length);
+        } else if (direction === 'prev') {
```

Comment:
Do we need a "else if" here? would there be another case except for "next" and "prev"?